pipelines/infra-ci-plan.yml

```diff
@@ -0,0 +1,37 @@
+# run a terraform plan against Dev for each PR
+
+trigger: none
+
+pr:
+  branches:
+    include:
+      - main

```

Comment:
Yes this means PRs that are merging into main. `*` would be fine, since we rarely merge into other branches anyway.

https://learn.microsoft.com/en-us/azure/devops/pipelines/repos/github?view=azure-devops&tabs=yaml#pr-triggers